src/main/java/org/osm2world/console/Output.java

```diff
@@ -58,6 +59,11 @@ public static void output(Configuration config,
 			CLIArgumentsGroup argumentsGroup)
 		throws IOException {
 
+		if (config instanceof PropertiesConfiguration) {
```

Comment:
Could be `instanceof PropertiesConfiguration propertiesConfig`

Reply:
Changed how I do that all together